homeassistant/components/media_source/__init__.py

```diff
@@ -189,7 +189,7 @@ async def websocket_resolve_media(
 ) -> None:
     """Resolve media."""
     try:
-        media = await async_resolve_media(hass, msg["media_content_id"])
+        media = await async_resolve_media(hass, msg["media_content_id"], None)
```

Comment:
This is not a correct fix. We should allow passing in an entity ID instead via the WS command.